docker/Dockerfile

```diff
@@ -32,5 +36,5 @@ EXPOSE ${CONTAINER_PORT}
 RUN alembic upgrade head
```

Comment:
If this is now happening on the startup script, do we really need it here?

Reply:
Also, we probably want to remove the `ENV` line as well... seems redundant with what the script is doing now.

Reply:
Will do.

Reply:
> Also, we probably want to remove the `ENV` line as well... seems redundant with what the script is doing now.

`ENV` line is needed to have some nice default values in case someone wants to use it in raw (as in non-configured) environments. This way, with a simple "docker run" backend should start with default values.

Reply:
But if these are defined there, the check from that script will always pass... and make no sense?